R/confidence_intervals.R

```diff
@@ -487,134 +663,272 @@ delta_method_ <- function(p,
-#'     and \code{distribution} manually.
-#'   }
-#' }
+#' and failure probabilities.
 #'
-#' @return A tibble containing the lifetime characteristic, the
-#'   probabilities, estimated standard errors by the delta method and computed
-#'   values for the specified confidence bound(s).
+#' @details
+#' The basis for the calculation of these confidence bounds are the standard errors
+#' determined by the \link[=delta_method]{delta method} and hence the required
+#' (log-)location-scale parameters as well as the variance-covariance matrix of
+#' these have to be estimated with \link[=ml_estimation]{maximum likelihood}.
 #'
-#' @export
-confint_fisher <- function(x, ...) {
-  UseMethod("confint_fisher")
-}
-
-#' Fisher Confidence Bounds for Quantiles and/or Probabilities
+#' The bounds on the probability are determined by the \emph{z-procedure}. See
+#' 'References' for more information on this approach.
 #'
-#' @inherit confint_fisher description return
+#' @inheritParams confint_betabinom
 #'
 #' @param x Object of class \code{model_estimation} returned from
 #'   \code{\link{ml_estimation}}.
-#' @inheritParams confint_betabinom.model_estimation
+#'
+#' @return A tibble with class \code{"confint"} containing the following columns:
+#'   \itemize{
+#'     \item \code{x} : An ordered sequence of the lifetime characteristic regarding
+#'       the failed units, starting at \code{min(x)} and ending up at \code{max(x)}.
+#'       With \code{b_lives = c(0.01, 0.1, 0.5)} the 1\%, 10\% and 50\% quantiles
+#'       are additionally included in \code{x}, but only if the specified
+#'       probabilities are in the range of the estimated probabilities.
+#'     \item \code{prob} : An ordered sequence of probabilities with specified
+#'       \code{b_lives} included.
+#'     \item \code{std_err} : Estimated standard errors with respect to
+#'       \code{direction}.
+#'     \item \code{lower_bound} : Provided, if \code{bounds} is one of
+#'       \code{"two_sided"} or \code{"lower"}. Lower limit of the confidence region
+#'       with respect to \code{direction}, i.e. quantiles or probabilities.
+#'     \item \code{upper_bound} : Provided, if \code{bounds} is one of
+#'       \code{"two_sided"} or \code{"upper"}. Upper limit of the confidence region
+#'       with respect to \code{direction}, i.e. quantiles or probabilities.
+#'     \item \code{distribution} : Specified distribution (determined when calling
+#'     \code{\link{ml_estimation}}).
+#'     \item \code{bounds} : Specified bound(s).
+#'     \item \code{direction} : Specified direction.
+#'     \item \code{method} : A character that is always \code{"conf_null"}. Due to
+#'       generic visualization functions column \code{method} has to be provided.
+#'   }
+#'
+#' @encoding UTF-8
+#'
+#' @references Meeker, William Q; Escobar, Luis A., Statistical methods for
+#'   reliability data, New York: Wiley series in probability and statistics, 1998
+#'
+#' @seealso \code{\link{confint_fisher.default}}
```

Comment:
\link{...} is needed. For the view I prefer \code{\link{...}}. 

I won't link confint_betabinom.* and confint_fisher* to each other. The latter is based on `ml_estimation` and hence `delta_method` is needed, whereas the other is based on rank theory and has nothing to do with MLE. Since the latter method is also the statistically accepted one, a link should not be used to `confint_betabinom*` and hence indirectly linking to `rank_regression`.